template/styles/layout.css

```diff
@@ -34,6 +34,14 @@
   transform: translateY(-50%);
 }
 
+.font-small {
+  font-size: 14px;
+}
+
+.font-large {
+  font-size: 18px;
+}
+
```

Comment:
I have to see the changes graphically in the slides, but worst case they can even be not used